pkg/apis/machineconfiguration.openshift.io/v1/types.go

```diff
@@ -358,6 +358,14 @@ const (
 
 	// MachineConfigPoolDegraded is the overall status of the pool based, today, on whether we fail with NodeDegraded or RenderDegraded
 	MachineConfigPoolDegraded MachineConfigPoolConditionType = "Degraded"
+
+	MachineConfigPoolBuildPending MachineConfigPoolConditionType = "BuildPending"
+
+	MachineConfigPoolBuilding MachineConfigPoolConditionType = "Building"
+
+	MachineConfigPoolBuildSuccess MachineConfigPoolConditionType = "BuildSuccess"
+
+	MachineConfigPoolBuildFailed MachineConfigPoolConditionType = "BuildFailed"
```

Comment:
If this makes it in before the API Migration does (it probably will) we'll just need to remember to stuff these into the API PR

Reply:
Noted.